.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: java
-jdk: openjdk8
 dist: trusty
+jdk:
```

Comment:
I would really like to have an IBM JVM here.. I am quite sure this is not so easily possible as just listing it here, but maybe some approach as Jonas had before with openjdk11? testing different versions is fine, but we also need to split on the providers (IBM, openjdk, etc).. We can have this in separate ticket, but this is important to have as early as possible as then we recognize problems right away..

Reply:
Done.